GrowthViz-pediatrics.py

```diff
@@ -539,26 +549,26 @@ def on_button_clicked(b):
 # 
 # The following visualizations show exclusions at age. Each plot shows a single exclusion type with the different runs shown in different colors.
 
-# In[37]:
+# In[ ]:
 
 
 combined['rounded_age'] = np.around(combined.age)
```

Comment:
I don't know if this changed in recent versions of one or more of the dependencies (seaborn? mpl?) but as is, this results in x-tick labels with a decimal point, which are hard to read. Adding `.astype(int)` at the end fixes this, e.g.:

`combined['rounded_age'] = np.around(combined_age).astype(int)`

Might need to make this change in both notebooks.